slsa/v1.0/L1/buildtype.md

```diff
@@ -0,0 +1,115 @@
+<!--
+SPDX-FileCopyrightText: 2022-2024 TII (SSRC) and the Ghaf contributors
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Build Type: Jenkins
+
+This is a [SLSA Provenance](https://slsa.dev/provenance/v1) `buildType` that describes the execution of a Jenkins workflow.
+
+This build type was inspired by the Cimon Jenkins build type described [here](https://docs.cimon.build/provenance/buildtypes/jenkins/v1).
+
+## Description
+
+```json
+"buildType": "https://github.com/tiiuae/ghaf-infra/blob/main/slsa/v1.0/L1/buildtype.md"
+```
+
+The Jenkins build type describes the execution of a Jenkins workflow that builds a software artifact.
```

Comment:
Is the "main artifact" the point here, or should this be plural "artifacts?"

Reply:
whether the build result counts as one or multiple artifacts is up to interpretation, but this is for one single target